[BUG]

## Environment
* `chromium`: `110`
* `puppeteer` / `puppeteer-core` Version: 
* Node.js Version: `14.21.2`
* Npm Version: `6.14.17`
* Lambda / GCF Runtime: `nodejs14.x`

## Expected Behavior

* Open chromium-browser properly in AWS Lambda NodeJS 14.x

## Current Behavior

* Failed creation of chromium

> * Received Error: _Cannot find module '/opt/nodejs/node_modules/@sparticuz/chromium/build/index.js'. Please verify that the package.json has a valid \"main\" entry_ 

## Steps to Reproduce

* Create chromium lambda layer by executing: 
``` js
// In Javascript
`git clone --depth=1 https://github.com/sparticuz/chromium.git && \
cd chromium && \
make chromium.zip && cp chromium.zip ${path.resolve(
    path.join(
      __dirname,
      '..',
      '..',
      'layers',
      'puppeteer',
      'chromium.zip'
    )
  )}`
```
* Upload the created chromium.zip to a layer through serverless framework
```
...
    package:
      artifact: layers/puppeteer/chromium.zip
    name: ${self:service}-puppeteer-layer-${self:provider.stage}
    description: Layer for puppeteer libraries
    compatibleRuntimes:
      - nodejs14.x
      - nodejs16.x
    retain: false
...
```
* The layer is correctly published but after looking into it, I notice there is no `build` folder inside `@sparticuz/chromium` folder.
* Also, I checked [release 110](https://www.npmjs.com/package/@sparticuz/chromium-min) and it does not include the build folder either. 

## Possible Solution

<!-- Not mandatory, but you can suggest a fix or reason for the bug. -->

--- a/Makefile
+++ b/Makefile
@@ -15,6 +15,7 @@ test:
 
 %.zip:
 	npm install --fund=false --package-lock=false
+	npm run build
 	mkdir -p nodejs
 	npm install --prefix nodejs/ tar-fs@2.1.1 --bin-links=false --fund=false --omit=optional --omit=dev --package-lock=false --save=false
 	npm pack

